components/ProfilePage/SmartWalletContents.tsx

```diff
@@ -26,8 +31,19 @@ const SmartWalletContents = () => {
     getDNAByCre8orNumber()
   }, [getDNAByCre8orNumber])
 
+  const [{ canDrop, isOver }, drop] = useDrop(() => ({
+    accept: ItemTypes.CRE8OR,
+    drop: (item) => {
+      console.log(item)
```

Comment:
do we need this `console.log` statement?

Reply:
```suggestion
```